carsus/io/vald/vald.py

```diff
@@ -193,38 +249,39 @@ def extract_linelist(self, vald):
                 vald linelist containing only the following columns:
                 atomic_number or chemical, ion_charge, wavelength, log_gf, rad, stark, waals
```

Comment:
e_low missing from this list

Reply:
Do we have a more complete description of what these values mean, too? This applies to other similar docstrings. They do not need to be in the docstrings, but *somewhere* in Carsus docs.

Reply:
Good catch. Updating the docstrings now. 

Where's the best place to describe what these values mean? I don't immediately know where makes the most sense.

Reply:
I would suggest adding a notebook to the datasets section of the docs with the info and possibly examples.